applications/connectivity_bridge/src/modules/usb_cdc_handler.c

```diff
@@ -0,0 +1,274 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+#include <stdio.h>
+#include <zephyr/types.h>
+#include <drivers/uart.h>
+#include <usb/usb_device.h>
+
+#define MODULE usb_cdc
+#include "module_state_event.h"
+#include "peer_conn_event.h"
+#include "cdc_data_event.h"
+#include "uart_data_event.h"
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(MODULE, CONFIG_BRIDGE_CDC_LOG_LEVEL);
+
+#define CDC_DEVICE_COUNT CONFIG_USB_CDC_ACM_DEVICE_COUNT
+#define CDC_DEVICE_NAME_TEMPLATE CONFIG_USB_CDC_ACM_DEVICE_NAME "_%d"
+
+#define USB_CDC_DTR_POLL_MS 500
+#define USB_CDC_RX_BLOCK_SIZE CONFIG_BRIDGE_BUF_SIZE
+#define USB_CDC_RX_BLOCK_COUNT (CDC_DEVICE_COUNT * 3)
+#define USB_CDC_SLAB_ALIGNMENT 4
+
+static void cdc_dtr_timer_handler(struct k_timer *timer);
+static void cdc_dtr_work_handler(struct k_work *work);
+
+K_TIMER_DEFINE(cdc_dtr_timer, cdc_dtr_timer_handler, NULL);
```

Comment:
Missing `static` for the timer, work and memslab (not sure about that though, can only find one place in NCS where `static` is used before `K_MEM_SLAB_DEFINE`)